cubes/query/browser.py

```diff
@@ -75,6 +76,12 @@
 _ReportResult = Union[AggregationResult, Facts, JSONType, List[JSONType]] 
 
 
+class BrowserFeatures(NamedTuple):
+    actions: Iterable[str]
```

Comment:
It would be better if this was an enum.

```python
class BrowserFeatureAction(Enum):
    aggregate = 1
    fact = 2
    facts = 3
    members = 4
    cell = 5
```

It requires proper string → value conversion through `BrowserFeatureAction[astring]` where converted from a dictionary.

I'm open for better name. I didn't go with `BrowserAction` intentionally, as it has stronger feel that the object actually represents some action that can be performed.

[Source](https://github.com/DataBrewery/cubes/blob/master/doc/extensions/backends.rst#browser-and-cube-features)